portfolio/src/main/webapp/script.js

```diff
@@ -41,15 +41,24 @@ function getComments() {
 
   fetch('/comments').then(response => response.json()).then((cmtContainer) => {
 
-    if(!cmtContainer) return;
+    if (!cmtContainer) return;
 
-    var str_comments = "<h2>Comments</h2>";
+    var commentsOutput = document.getElementById("comments-output");
```

Comment:
nit: Use `const` when you don't need to redefine the variable later (or `let` if you do) https://google.github.io/styleguide/jsguide.html#features-use-const-and-let